packages/playground/src/components/pipeline-studio/pipeline-studio-footer-bar/pipeline-studio-footer-bar.tsx

```diff
@@ -8,7 +8,7 @@ interface PipelineStudioFooterBarProps {
     info: number
   }
   commitHistory: {
-    lastCommittedAt: number
+    lastCommittedAt: string
```

Comment:
This is to ensure playground components always get the final formatted time string in human readable format, to ensure consistency across the application.